models/bamf_nnunet_ct_liver/meta.json

```diff
@@ -0,0 +1,137 @@
+{
+    "id": "",
+    "name": "bamf_nnunet_ct_liver",
+    "title": "AIMI CT Liver",
+    "summary": {
+        "description": "An nnU-Net based model to segment liver from CT scans",
+        "inputs": [
+            {
+                "label": "Input Image",
+                "description": "The CT scan of a patient.",
+                "format": "DICOM",
+                "modality": "CT",
+                "bodypartexamined": "LIVER",
```

Comment:
Please change to "Liver" (I would've done it but to edit files the checkbox that gives repo owners access to the fork needs to be set).